### ISSUE TITLE
Missing Build.sh

### ISSUE THREAD

**Issue by brainlessbitch** [NerdyPepper/scientifica#64]
Where is build.sh??

**Comment by oppiliappan** [comment]
`build.sh` was a bit of a hackjob. builds for scientifica are now done with nix; see `flake.nix`. the install docs need to be updated. creating a build of scientifica is now just:

```bash
# with nix installed

$ git clone https://github.com/nerdypepper/scientifica
$ nix build

# built font files and stored in the ./result directory
```

**Comment by oppiliappan** [comment]
if you would still like to use `build.sh` you can find the script in parent commits of https://github.com/nerdypepper/scientifica/commit/6b31fa5d24df2949c7987f6fa349e6ef86e2fc00.

**Comment by brainlessbitch** [comment]
> `build.sh` was a bit of a hackjob. builds for scientifica are now done with nix; see `flake.nix`. the install docs need to be updated. creating a build of scientifica is now just:
> 
> ```shell
> # with nix installed
> 
> $ git clone https://github.com/nerdypepper/scientifica
> $ nix build
> 
> # built font files and stored in the ./result directory
> ```

i don’t know much about nix, can you install it alongside other package managers? (in my case pacman)

**Comment by oppiliappan** [comment]
yes absolutely, please see https://nixos.org/download for more. you can install just `nix` (the package manager, as opposed to `nixos`, the operating system), and use it for builds. 

is there a specific reason you require a manual build of the font? if not, you can download the v2.3 release from the [releases](https://github.com/nerdypepper/scientifica/releases/tag/v2.3) page.
